common/religion/religions/secular_group.txt

```diff
@@ -75,7 +75,7 @@
 	}
 
 	holy_order_names = {
-		{ name = "holy_order_secular" coat_of_arms = "ho_secular" }
+		{ name = "holy_order_secular" }
```

Comment:
Need more holy orders. It may be not enough.

Reply:
Added a few more orders names.